daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

```diff
@@ -160,7 +163,7 @@ Dapr can use any Redis instance - containerized, running on your local dev machi
 
 *Note: Dapr does not support Redis >= 7. It is recommended to use Redis 6*
 
-{{< tabs "Self-Hosted" "Kubernetes" "AWS" "GCP" "Azure">}}
+{{< tabs "Self-Hosted" "Kubernetes" "AWS" "GCP" "Azure Cache for Redis">}}
```

Comment:
If you are using the full product name for Azure, then we should do the same for AWS (https://docs.aws.amazon.com/whitepapers/latest/scale-performance-elasticache/elasticache-for-redis.html) and GCP. Do you want to have the full product name?

Reply:
I'll leave it as Azure, but will rearrange to be alphabetical